### PR TITLE
Website: Update react recipe to latest cra-demo readme

### PR DESCRIPTION
**Pull request by micahgodbolt**
Update demo to use current cra-demo code and readme content

### CODE REVIEW

File: docs/content/docs/recipes/react.md [hunk 1]
```diff
@@ -5,247 +5,211 @@ aliases:
   - "/start/react-tutorial/"
 ---
 
-In this tutorial, you'll learn about using the Fluid Framework by building a simple application that enables every client of the application to change a dynamic time stamp on itself and all other clients almost instantly. You'll also learn how to connect the Fluid data layer with a view layer made in [React](https://reactjs.org/). The following image shows the time stamp application open in four browsers. Each has a button labelled **click** and beside it a UNIX Epoch time. The same time in all four. The cursor is on the button in one browser.
+In this tutorial, you'll learn about using the Fluid Framework by building a simple application that enables every client of the application to change a dynamic time stamp on itself and all other connected clients. You'll also learn how to connect the Fluid data layer with a view layer made in [React](https://reactjs.org/).
```
Comment (tylerbutler) [inline docs/content/docs/recipes/react.md:8]: I prefer the old wording.

File: docs/content/docs/recipes/react.md [hunk 2]
```diff
@@ -5,247 +5,211 @@ aliases:
...
 
-![Four browsers with the Timestamp app open in them.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-1.PNG)
+The following image shows finished application loaded into four browsers. Each browser contains a button and a timestamp. When the button is pressed in any of the browsers, the timestamp is updated in all four browsers.
 
-The following image shows the same four clients one second after the **click** button was pressed. Note that the timestamp has updated to the very same time in all four browsers.
-
-![Four browsers with the Timestamp app open in them one second after the button has been pushed.](https://fluidframework.blob.core.windows.net/static/images/Four-clients-2.PNG)
+![cra](https://user-images.githubusercontent.com/1434956/111496992-faf2dc00-86fd-11eb-815d-5cc539d8f3c8.gif)
```
Comment (tylerbutler) [inline docs/content/docs/recipes/react.md:12]: No gifs please. Accessibility problems. Also we should upload images to our CDN.

File: docs/content/docs/recipes/react.md [hunk 3]
```diff
@@ -5,247 +5,211 @@ aliases:
...
-    |fluidframework/azure&#x2011;client&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   |Defines the connection to a Fluid service server and defines the starting schema for the [Fluid container][]. In this tutorial, we will use a local test service called Tinylicious.|
-    &nbsp;
+In this example you will do the following:
 
-    Run the following command to install the libraries.
+  - [Use Create React App](use-create-react-app)
+  - [Install Fluid package dependencies](install-fluid-package-dependencies)
+  - [Import and initialize Fluid dependencies](import-and-initialize-fluid-dependencies)
+  - [Get the Fluid SharedMap](get-the-fluid-sharedmap)
+  - [Update the view](update-the-view)
```
Comment (tylerbutler) [inline docs/content/docs/recipes/react.md:37]:
```suggestion
  - [Use Create React App](#use-create-react-app)
  - [Install Fluid package dependencies](#install-fluid-package-dependencies)
  - [Import and initialize Fluid dependencies](#import-and-initialize-fluid-dependencies)
  - [Get the Fluid SharedMap](#get-the-fluid-sharedmap)
  - [Update the view](#update-the-view)
```

File: docs/content/docs/recipes/react.md [hunk 4]
```diff
@@ -5,247 +5,211 @@ aliases:
...
-1. Open a Command Prompt and navigate to the parent folder where you want to create the project; e.g., `c:\My Fluid Projects`.
-1. Run the following command at the prompt. (Note that the CLI is np**x**, not npm. It was installed when you installed Node.js.)
 
-    ```dotnetcli
-    npx create-react-app fluid-react-tutorial --use-npm
-    ```
+\* Just want to see the code? Jump to the [finished tutorial.](https://github.com/microsoft/FluidExamples/blob/main/cra-demo/src/App.js).
 
-1. The project is created in a subfolder named `fluid-react-tutorial`. Navigate to it with the command `cd fluid-react-tutorial`.
-1. The project uses two Fluid libraries:
```
Comment (tylerbutler) [inline docs/content/docs/recipes/react.md:32]: Why remove this?

**Review by tylerbutler — COMMENTED** [review]
I'm a little confused about all the removals. Was the source code changed so dramatically that all the prose had to be rewritten? I feel like you also changed the target "level" of the tutorial. It was 100-level, but now I think it's more 200-level.

File: docs/content/docs/recipes/react.md (hunk 2 above)
Comment (micahgodbolt) [reply docs/content/docs/recipes/react.md:12]: ok, i'll return the top portion to be the same

File: docs/content/docs/recipes/react.md (hunk 3 above)
Comment (micahgodbolt) [reply docs/content/docs/recipes/react.md:40]: weird.....i wonder why those dissapeared

Comment (micahgodbolt) [reply docs/content/docs/recipes/react.md]: yeah, broken in example repo. fixed there.

File: docs/content/docs/recipes/react.md (hunk 4 above)
Comment (micahgodbolt) [reply docs/content/docs/recipes/react.md]: it was older content. the new demo has the NPX command lower down.

Comment (micahgodbolt) [reply docs/content/docs/recipes/react.md]: i was able to return much of this content as it doesn't conflict with the new approach

**Comment by micahgodbolt** [comment]
Huge chunks needed to be removed and rewritten due to removal of azure client, removal of getClientId, and change in organization of functionality. I agree that the voicing is different between this tutorial and the cra-demo readme.

**Pull request closed without merging**